docs/img/ghost.svg

```diff
@@ -1,96 +1,96 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
-
-<svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
-   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
-   width="40"
-   height="40"
-   viewBox="0 0 10.583333 10.583334"
-   version="1.1"
-   id="svg821"
-   inkscape:version="0.92.3 (2405546, 2018-03-11)"
-   sodipodi:docname="ghost.svg">
-  <defs
-     id="defs815" />
-  <sodipodi:namedview
-     id="base"
-     pagecolor="#ffffff"
-     bordercolor="#666666"
-     borderopacity="1.0"
-     inkscape:pageopacity="0.0"
-     inkscape:pageshadow="2"
-     inkscape:zoom="2.8"
-     inkscape:cx="-47.170224"
-     inkscape:cy="24.659748"
-     inkscape:document-units="mm"
-     inkscape:current-layer="layer1"
-     showgrid="false"
-     units="px"
-     inkscape:pagecheckerboard="true"
-     inkscape:window-width="1680"
-     inkscape:window-height="988"
-     inkscape:window-x="-8"
-     inkscape:window-y="-8"
-     inkscape:window-maximized="1" />
-  <metadata
-     id="metadata818">
-    <rdf:RDF>
-      <cc:Work
-         rdf:about="">
-        <dc:format>image/svg+xml</dc:format>
-        <dc:type
-           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
-        <dc:title></dc:title>
-      </cc:Work>
-    </rdf:RDF>
-  </metadata>
-  <g
-     inkscape:label="Calque 1"
-     inkscape:groupmode="layer"
-     id="layer1"
-     transform="translate(0,-286.41665)">
-    <rect
-       style="fill:#4d4d4d;fill-rule:evenodd;stroke-width:0.29778239"
-       id="rect1366"
-       width="5.5042777"
-       height="1.8520833"
-       x="0"
-       y="286.41666" />
-    <rect
-       style="fill:#4d4d4d;fill-rule:evenodd;stroke-width:0.41291448"
-       id="rect1366-7"
-       width="10.583333"
-       height="1.8520833"
-       x="0"
-       y="290.78229" />
-    <rect
-       y="286.41666"
-       x="7.8430061"
-       height="1.8520833"
-       width="2.7875741"
-       id="rect1389"
-       style="fill:#4d4d4d;fill-rule:evenodd;stroke-width:0.21191517" />
-    <rect
-       y="295.14789"
-       x="-10.583333"
-       height="1.8520833"
-       width="5.5042777"
-       id="rect1391"
-       style="fill:#4d4d4d;fill-rule:evenodd;stroke-width:0.29778239"
-       transform="scale(-1,1)" />
-    <rect
-       style="fill:#4d4d4d;fill-rule:evenodd;stroke-width:0.21191517"
-       id="rect1393"
-       width="2.7875741"
-       height="1.8520833"
-       x="-2.7875741"
-       y="295.14789"
-       transform="scale(-1,1)" />
-  </g>
-</svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Changing this resource location will break the UI, won't it ?

Reply:
Seems to be unused, any references into src or public folders (exploited svg are located & defined directly into `src/js/components/commons/icons`)